lib/plugins/entities.js

```diff
@@ -975,7 +1000,13 @@ function inject (bot) {
   }
 
   function fetchEntity (id) {
-    return bot.entities[id] || (bot.entities[id] = new Entity(id))
+    if (!bot.entities[id]) {
+      const entity = new Entity(id)
+      entity.passengers = []
```

Comment:
why not put this directly in prismarine entity ?

if we do that here why do we need to check passengers is an array above?